src/api/app/assets/javascripts/webui2/kiwi_editor.js

```diff
@@ -0,0 +1,448 @@
+var canSave = false;
```

Comment:
This is a lot of JavaScript. :see_no_evil: I know we have a lot of Javascript in the old webui for this too, but maybe Bootstrap can help us to reduce it. :thinking:

Reply:
I will try my best. I actually just found out there is a common function for autocomplete, so it certainly will be shorter.